.github/workflows/py-hypertrace.yml

```diff
@@ -1,91 +0,0 @@
-name: Hypertrace
-
-on: [push, pull_request]
-
-jobs:
-  sRTMnet:
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: bash -l {0}
-    container:
-      image: jammont/isofit:amd64
-      options: --user root
-
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-
-    - name: Enable ISOFIT Environment
-      run: |
-        echo PATH=/home/ray/anaconda3/envs/isofit/bin/:$PATH >> $GITHUB_ENV
-
-    - name: Install Isofit
-      run: |
-        pip install dask netCDF4
-        pip install 'xarray<2024.1.1' --force-reinstall
-        pip install --no-deps --force-reinstall --editable .
-
-    - name: Download and extract hypertrace data
-      run: |
-        cd examples/py-hypertrace
-        wget https://github.com/ashiklom/isofit/releases/download/hypertrace-data/hypertrace-data.tar.gz
-        tar -xf hypertrace-data.tar.gz
-
-    - name: Run Hypertrace
-      run: |
-        cd examples/py-hypertrace
-        # Modify the workflow file for GitHub actions
-        python workflow.py configs/example-srtmnet-ci.json
-        python summarize.py configs/example-srtmnet-ci.json
-
-    - name: Upload summary from CI workflow
-      uses: actions/upload-artifact@v2
-      with:
-        name: srtmnet-summary
-        path: examples/py-hypertrace/output/example-srtmnet/summary.csv
```

Comment:
I have not yet moved this behavior to the new `pip` package testing workflow. Is it important that this artifact be uploaded? It will need to be uploaded for each item in the test matrix – currently once per Python version.